crnk-example-service/src/main/java/io/crnk/example/service/model/MovieEntity.java

```diff
@@ -57,5 +60,55 @@
 
 	@Version
 	private Integer version;
+	
+    //This is an example on links customization.
+	//The links object in the response will look like
+	//"links":{"rates":"https://www.imdb.com/{MovieID}/rates", "comments":"https://www.imdb.com/{MovieID}/comments" }
+	//You can find that there is no self link type here, If you want to add it beside your customization just remove the comments below.
+    @Transient
+    @JsonApiLinksInformation
+    private MovieLinks links = new MovieLinks();
+
+    public class MovieLinks implements  LinksInformation//,SelfLinksInformation
```

Comment:
implements SelfLinksInformation would seems right.

Reply:
Lombok @Data would allow you to get rid of the setters

Reply:
done if u have any more comments please specify it : ) Thanks!